components/automate-ui/src/app/pages/user-details/user-details.component.scss

```diff
@@ -4,6 +4,7 @@
   display: flex;
   justify-content: space-between;
   padding: 0 0 25px 0;
+  height: 115px;
```

Comment:
CSS changes to preserve the page header height even when heading values are `nil`

Reply:
Nice--and thank you for the comment!!